bodhi/tests/server/consumers/test_masher.py

```diff
@@ -1296,16 +1296,19 @@ def test_absent_gating(self, publish, *args):
     def test_modify_testing_bugs(self, on_qa, modified, *args):
         self.masher.consume(self._make_msg())
 
-        expected_message = (
+        expected_message_start = (
             u'bodhi-2.0-1.fc17 has been pushed to the Fedora 17 testing repository. If problems '
             u'still persist, please make note of it in this bug report.\nSee '
             u'https://fedoraproject.org/wiki/QA:Updates_Testing for\ninstructions on how to '
             u'install test updates.\nYou can provide feedback for this update here: {}')
-        expected_message = expected_message.format(
+        expected_message_start = expected_message_start.format(
             urlparse.urljoin(
                 config['base_address'],
-                '/updates/FEDORA-{}-a3bbe1a8f2'.format(datetime.datetime.now().year)))
-        on_qa.assert_called_once_with(12345, expected_message)
+                '/updates/FEDORA-{}-'.format(datetime.datetime.now().year)))
+        called_args = on_qa.mock_calls[0][1]
```

Comment:
Hmm, I'm not sure what is going on here. It had seemed that the update ID was being generated in a predictable manner in the test suite before. Perhaps the predictable manner depends on the Python version (like Python 2 predicably makes this, but Python 3 makes something else?).

I'll see if I can find what is making these strings in the test suite.

Reply:
I found it!

Remember that thing that was patching ```uuid.uuid4``` with "wat"? [```Update.assign_alias()```](https://github.com/fedora-infra/bodhi/blob/3.4.0/bodhi/server/models.py#L2238-L2249) was using that to basically make a predictable update alias. So if we can bring that mock back I think this string will be deterministic and identical to what we see here.